homeassistant/components/zha/core/cluster_handlers/general.py

```diff
@@ -347,26 +348,10 @@ def __init__(self, cluster: zigpy.zcl.Cluster, endpoint: Endpoint) -> None:
         super().__init__(cluster, endpoint)
         self._off_listener = None
 
-        if self.cluster.endpoint.model not in (
-            "TS011F",
-            "TS0121",
-            "TS0001",
-            "TS0002",
-            "TS0003",
-            "TS0004",
-        ):
-            return
-
-        try:
-            self.cluster.find_attribute("backlight_mode")
-        except KeyError:
-            return
```

Comment:
Review note:
This should no longer be needed, since all devices with a matching quirk ID (in this case with `TUYA_PLUG_ONOFF`) should be guaranteed to have the custom attributes defined correctly.